transport/control.go

```diff
@@ -56,43 +56,33 @@ type windowUpdate struct {
 	increment uint32
 }
 
-func (windowUpdate) isItem() bool {
-	return true
-}
+func (*windowUpdate) isItem() {}
```

Comment:
I think this is better as-is. Unfortunately you lose too much type safety when you implement interfaces with values in Go.